docs/releases/2.2.txt

```diff
@@ -46,6 +46,10 @@ Minor features
 
 * Added a CSS class to the column headers of
   :class:`~django.contrib.admin.TabularInline`.
+* Added ``django.contrib.admin.EmptyFieldListFilter`` in
```

Comment:
Add a new line. Not sure the wording is correct here, it wasn't added **in** `list_filter` but it should be used in `list_filter`?

Reply:
maybe use "for" instead of "in", or just get rid of the `ModelAdmin.list_filter` reference?